packages/tests/esm/src/jest.d.ts

```diff
@@ -0,0 +1,14 @@
+// These imports are gross, but typescript won't compile if they're just coded normally
```

Comment:
Again, copied from `integration`. We might make this nicer for new users in the future. See https://github.com/joist-orm/joist-orm/issues/1622